src/pages/dar_application/ResearcherInfo.js

```diff
@@ -1,11 +1,11 @@
+// noinspection SqlDialectInspection
```

Comment:
Is this a JetBeans thing? This doesn't need to be in the code.

Reply:
Yes, it's a Intellij thing. Intellij flags the long "Select ... " line as a sql statement.